packages/mgt-components/src/components/mgt-person-card/mgt-person-card.ts

```diff
@@ -1187,6 +1220,37 @@ export class MgtPersonCard extends MgtTemplatedComponent {
     return person && person.personImage ? person.personImage : null;
   }
 
+  private hovered(icon: string, state: boolean) {
+    switch (icon) {
```

Comment:
This works great, but it's not easily extended. 
If you were to use a `Record<string,boolean>` then adding a new thing that needs a hover state to be tracked like this becomes much easier in the future, and way less code.

const hoverStates: Record<string, boolean> = {}

private setHoveredState(icon: string, state: boolean) {
  this.hoverStates[icon] = state;
}

private getHoverState(icon: string) {
  return this.hoverStates.hasOwnProperty(icon)? this.hoverStates[icon] : false;
}